src/scalable_cuckoo_filter.rs

```diff
@@ -192,6 +192,21 @@ impl<T: Hash + ?Sized, H: Hasher + Clone, R: Rng> ScalableCuckooFilter<T, H, R>
         self.filters.iter().map(|f| f.bits()).sum()
     }
 
+    /// Returns the false positive probability.
+    pub fn false_positive_probability(&self) -> f64 {
+        self.false_positive_probability
+    }
+
+    /// Returns the number of elements in each buckets.
+    pub fn bucket_size(&self) -> usize {
```

Comment:
How about renaming this method to `entries_per_bucket`?
(Because the value is set via [ScalableCuckooFilterBuilder::entries_per_bucket()](https://docs.rs/scalable_cuckoo_filter/latest/scalable_cuckoo_filter/struct.ScalableCuckooFilterBuilder.html#method.entries_per_bucket) method, it feels more natural to use the same name for this getter method.)

```suggestion
    pub fn entries_per_bucket(&self) -> usize {
```